<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewTest</class>
 <widget class="QWidget" name="CreateNewTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateNewTest</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Question</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="questionEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Course</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabs">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>461</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="open">
    <attribute name="title">
     <string>Open</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="answerOpenEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="close">
    <attribute name="title">
     <string>Close</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rbA">
            <property name="text">
             <string>a)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <widget class="QRadioButton" name="rbB">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>33</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>b)</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <widget class="QRadioButton" name="rbC">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>13</y>
            <width>32</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>c)</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <widget class="QRadioButton" name="rbD">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>13</y>
            <width>33</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>d)</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" rowspan="2" colspan="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTextEdit" name="answerCloseAEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="answerCloseBEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QTextEdit" name="answerCloseCEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QTextEdit" name="answerCloseDEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Answer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="help">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="back">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="next">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="questionNumber">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>33</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>questionEdit</zorder>
  <zorder>save</zorder>
  <zorder>tabs</zorder>
  <zorder>label_2</zorder>
  <zorder>help</zorder>
  <zorder>questionNumber</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>next</tabstop>
  <tabstop>back</tabstop>
  <tabstop>save</tabstop>
  <tabstop>questionEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>next</sender>
   <signal>clicked()</signal>
   <receiver>CreateNewTest</receiver>
   <slot>nextQuestion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back</sender>
   <signal>clicked()</signal>
   <receiver>CreateNewTest</receiver>
   <slot>backQuestion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabs</sender>
   <signal>currentChanged(int)</signal>
   <receiver>CreateNewTest</receiver>
   <slot>currentChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save</sender>
   <signal>clicked()</signal>
   <receiver>CreateNewTest</receiver>
   <slot>saveCourse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>nextQuestion()</slot>
  <slot>backQuestion()</slot>
  <slot>currentChangedSlot()</slot>
  <slot>writeToCloseB()</slot>
  <slot>saveCourse()</slot>
 </slots>
</ui>
